SYCL/InlineAsm/asm_float_add_aspect-fp64.cpp

```diff
@@ -0,0 +1,11 @@
+// Enable FP64 part of <asm_float_add.cpp>. To be removed once DPC++
+// supports optional device features and the code could be enabled
+// unconditionally without causing failures in speculative compilation
+// of the kernels.
```

Comment:
I don't think this is accurate. Removal of the ENABLE_FP64 would result in dropping one of the cases in the main file, not executing them both.